### ISSUE TITLE
Hardware Version 4 schematic still has CAN2-B crossed at K2 

### ISSUE THREAD

**Issue by traviswparker** [Heavy-Vehicle-Networking-At-U-Tulsa/can-man-in-the-middle#7]
The hardware version 4 schematic still has relay K2 connecting CAN2L-A to CAN2H-B and CAN2H-A to CAN2L-B. This is the fault that breaks passthrough on version 3.

**Comment by jeremydaily** [comment]
Agreed. There is a hardware version that has been rolled already, so I'll have to post an errata in this directory and look to design a Rev 5 this summer. 

Any feature requests?
